cpp/oneapi/dal/detail/common.hpp

```diff
@@ -141,7 +141,7 @@ constexpr bool is_floating_point() {
 }
 
 template <typename Data>
-struct integer_overflow_ops {
+struct ONEDAL_EXPORT integer_overflow_ops {
```

Comment:
Export is not needed here, for template classes export takes effect only when it appears in template instantiation or specialization